wgengine/router/router_linux.go

```diff
@@ -165,23 +166,28 @@ func (r *linuxRouter) Close() error {
 
 // Set implements the Router interface.
 func (r *linuxRouter) Set(cfg *Config) error {
+	errors := []error(nil)
```

Comment:
Don't name it "errors", as that's the name of https://golang.org/pkg/errors/ and we don't want to shadow it.

And style-wise, just:

```go
   var errs []error
```

... is fine.